addons/project/static/src/views/burndown_chart/burndown_chart_search_model.js

```diff
@@ -93,10 +99,10 @@ export class BurndownChartSearchModel extends SearchModel {
      * @param fieldName The field name(s) the notification has to be related to.
      * @private
      */
-    _addGroupByNotification(fieldName) {
-        const notif = _t("The Burndown Chart must be grouped by");
+    _addGroupByNotification(body) {
+        const notif = _t(body);
```

Comment:
The content of a gettext call should always be a static string. This is not translatable ⚠